src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const inputDate = date.split(fromFormat[3]);
+  const number = inputDate[fromFormat.indexOf('DD')];
+  const month = inputDate[fromFormat.indexOf('MM')];
+  const year = inputDate[fromFormat.indexOf('YYYY')]
+  || inputDate[fromFormat.indexOf('YY')];
+
+  const copyToFormat = [...toFormat];
+  const dateBreakPoint = 30;
+  const currentCentury = 20;
+  const lastCentury = 19;
```

Comment:
I mean, this is unnecessary cause you used them only once and probably they won't change, but okay